test/helpers/populateHash.test.js

```diff
@@ -1,73 +0,0 @@
-import { assert } from 'chai'
-import { populateHash } from '../../src/helpers'
-
-describe('populateHash helper function', () => {
-  it('replaces `target` empty keys with `source` ones', (done) => {
-    const source = { key1: 'value1' }
-    const target = { key1: '' }
-    const res    = populateHash(source, target)
-
-    assert.deepEqual(res, { key1: 'value1' })
-    done()
-  })
-
-  it('leaves untouched `target` keys that are not empty', (done) => {
```

Comment:
Some of these tests should be ported then don't you think?

Reply:
I have gone through them and they looked very similar if not the same as those in `mergeHashes.test.js`. `mergeHashes` and `populateHash` did basically the same thing except that the latter one would merge only if the target was an empty string, which we said is not necessary. Is there something you would add?

Reply:
One of my comments wasn't published but with the latest commit, I think we are good.

Reply:
I'm not a git expert, but master and origin/master are ancestors of the branch I created. Why do you think it's not rebased?

Reply:
Because there are 15 commits and some of them are rewrite some of my own commits that are already in master. I can take care of the rebase.